lib/sdf-server/src/server/service/v2/variant.rs

```diff
@@ -122,32 +188,148 @@ pub async fn get_variant(
 
     Ok(Json(schema_variant))
 }
-#[remain::sorted]
-#[derive(Debug, Error)]
-pub enum SchemaVariantsAPIError {
-    #[error("schema error: {0}")]
-    Schema(#[from] dal::SchemaError),
-    #[error("schema error: {0}")]
-    SchemaVariant(#[from] dal::SchemaVariantError),
-    #[error("transactions error: {0}")]
-    Transactions(#[from] dal::TransactionsError),
-}
 
-impl IntoResponse for SchemaVariantsAPIError {
-    fn into_response(self) -> Response {
-        let status_code = match &self {
-            Self::Transactions(dal::TransactionsError::BadWorkspaceAndChangeSet) => {
-                StatusCode::FORBIDDEN
+#[instrument(
+    name = "sdf.v2.variant.determine_can_update_many",
+    level = "debug",
+    skip_all
+)]
+async fn determine_can_update_many(
```

Comment:
Cannot go in the `dal` since `module-index-client` should only be accessible by `sdf`. Didn't want to make a new module since the `v2` stuff is brand new and things are settling.